companies-manifest.js

```diff
@@ -647,6 +648,27 @@ const companies = [
     },
     projects: {},
   },
+  {
+    key: 'https://chemondis.com/',
+    similarWebGlobalRank: 404819,
+    name: 'CheMondis',
+    logo: CheMondisLogo,
+    style: {
+      bottom: '-0.2rem',
+    },
+    projects: {
+      marketplace: {
+        title: 'Leading B2B Marketplace for chemicals',
```

Comment:
I don't want to feel too nitpicky now so I'm sorry if I sound like that 😔
This title feels a little more like a tagline than a title... if you can't think of anything else then I would ignore my previous comment and just go with "CheMondis Marketplace"
Again, sorry for the conflicting opinions, I'm trying to get a grasp of how we should do it moving on too :)